src/main/resources/templates/manager/detail.html

```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
       xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
     <head>
-        <title>Spring Security Example </title>
+        <title>Details</title>
```

Comment:
Similar to the employee details page, the link to the edit page should be /employee/edit and needs to be changed for external user as well.